setup.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import setuptools
+
+if __name__ == "__main__":
+    setuptools.setup(
+        name="omega-moderne-client",
+        version="0.0.1",
+        author="Jonathan Leitschuh",
```

Comment:
Suggestion: extract your information and set to environment variables. This way, whoever using these public scripts is not impersonating you by design.

Reply:
I mean, I am the author of this particular project. So in this place, it's relevant as it will get bundled with the Egg. But in general, I'll eventually scrape my name out of this repository and replace it with more ao-centric names